packages/utils/package.json

```diff
@@ -29,7 +29,8 @@
     "compare-resource-counts-per-file": "bash src/fhir-converter/scripts/compare_resource_counts_per_file.sh",
     "compare-total-resource-count-post-fhir-insert": "bash src/fhir-converter/scripts/compare_total_resource_count_post_fhir_insert.sh",
     "uuidv7": "ts-node src/make-uuidv7",
-    "saml-server": "ts-node src/saml/saml-server"
+    "saml-server": "ts-node src/saml/saml-server",
+    "load-test": "artillery run ./src/terminology/load-test.yml"
```

Comment:
Why are we not adding this to the terminology's folder? We have testing for the FHIR Converter on the utils and I think this is not the way to go, it makes the utils package bulkier, it exposes things from the terminology to the utils (increases dependencies).